src/main/kotlin/com/avail/dispatch/InternalLookupTree.kt

```diff
@@ -117,7 +112,7 @@ internal constructor(
 	 * about this node's [argumentTypeToTest], [argumentPositionToTest], and
 	 * [ifCheckHolds], and [ifCheckFails].
 	 */
-	internal fun expandIfNecessary(
+	internal fun <AdaptorMemento>expandIfNecessary(
```

Comment:
All of these generic functions should have a space before the method name.

Reply:
Done, for all Kotlin files.